README.md

```diff
@@ -53,15 +53,19 @@ module.exports = {
     loaders: [
       {
         test: /\.js$/,
-        loader: 'jsx-loader?harmony'
+        loader: 'babel'
       },
       {
-        test: /\.(css)(\?.+)$/,
-        loader: ExtractTextPlugin.extract('style-loader', 'css-loader')
+        test: /\.css$/,
+        loader: ExtractTextPlugin.extract('style', 'css')
```

Comment:
I'm not sure this is safe to drop `-loader` suffix. If for example `css` package is installed then `'css'` will resolve to that instead of `css-loader`.